sdk/storage/tests.yml

```diff
@@ -18,10 +18,9 @@ stages:
         AllocateResourceGroup: 'false'
         BuildTargetingString: ${{ service }}
         ${{ if eq(service, 'azure-storage-file-datalake') }}:
-          MaxParallel: 1
-        ${{ if ne(service, 'azure-storage-file-datalake') }}:
+          DeployArmTemplate: true
```

Comment:
Where are we specifying where to fetch the ARM template from?

Reply:
this script will look for test-resources.json and deploy
https://github.com/Azure/azure-sdk-for-python/blob/f3760fc8d7ea1b46b0def0628579d36abe75976f/eng/common/TestResources/New-TestResources.ps1